nereid/nereid/main.py

```diff
@@ -1,8 +1,36 @@
 from fastapi import FastAPI
+from fastapi.openapi.docs import (
+    get_redoc_html,
+    get_swagger_ui_html,
+    get_swagger_ui_oauth2_redirect_html,
+)
+from starlette.staticfiles import StaticFiles
 
 from nereid.api.api_v1.api import api_router
 from nereid.core import config
 
-app = FastAPI()
+app = FastAPI(title="nereid", docs_url=None, redoc_url=None)
+app.mount("/static", StaticFiles(directory="nereid/static"), name="static")
+
+
+@app.get("/docs", include_in_schema=False)
+async def custom_swagger_ui_html():
+    return get_swagger_ui_html(
```

Comment:
My understanding is that an `async` needs a corresponding `await`. Is that in the works?

Reply:
I believe that `async def` functions use return just the same way regular `def` functions do. The 'await' statement is used within an async function if you need to get the value returned by another async function before returning the function you're in (oy, that's a mouthful).

e.g.,:
```python
async def custom_swagger_ui_html():
    some_intermediate_value = await the_return_from_some_other_async_function()
    return get_swagger_ui_html(value=some_intermediate_value)
```

in an async def function we would await the value from another async function, but we don't return our value using await.

Note that this is super hard to test against since python doesn't really have the tools to tell you if your async function is actually sync, and is running blocking code... in my experience, i can async whatever i want and things will continue to 'just work'. However, this case was copied from the actual FastAPI's [documentation ](https://fastapi.tiangolo.com/advanced/extending-openapi/#disable-the-automatic-docs)written by the author.

Reply:
Correction, you may use `await` to return the result from the enclosing function, but in this case they are equivalent (real citation needed, i just read a few tutorials). It seems to me that it's only important to use await if the operation is internal to the coroutine or if the return value is actually a coroutine itself.